Makefile

```diff
@@ -178,7 +178,7 @@ build: buildsrc gen
 buildsrc: crypto/libs/$(OS_TYPE)/$(ARCH)/lib/libsodium.a node_exporter NONGO_BIN deps $(ALGOD_API_SWAGGER_INJECT) $(KMD_API_SWAGGER_INJECT)
 	mkdir -p tmp/go-cache && \
 	touch tmp/go-cache/file.txt && \
-	GOCACHE=$(SRCPATH)/tmp/go-cache go install $(GOTRIMPATH) $(GOTAGS) $(GOBUILDMODE) -ldflags="$(GOLDFLAGS)" ./...
+	GOCACHE=$(SRCPATH)/tmp/go-cache && go install $(GOTRIMPATH) $(GOTAGS) $(GOBUILDMODE) -ldflags="$(GOLDFLAGS)" ./...
```

Comment:
I think that you'll need to make it into a "export GOCACHE=..." if you want to add the &&

Reply:
Didn't have any issue but I'll do because it works better with `&&`